pkg/server/container_create.go

```diff
@@ -206,6 +217,33 @@ func (c *criContainerdService) CreateContainer(ctx context.Context, r *runtime.C
 			specOpts = append(specOpts, apparmor.WithProfile(strings.TrimPrefix(appArmorProf, profileNamePrefix)))
 		}
 	}
+
+	// Set seccomp profile
```

Comment:
We should probably move this into a separate function. I'm fine with doing that in another PR.

Reply:
yeah it's almost the same.. I figured there may be a reason for the logic to split but it didn't so far.

Reply:
todo added